robottelo/hosts.py

```diff
@@ -854,22 +854,39 @@ def put_ssh_key(self, source_key_path, destination_key_name):
             raise CLIFactoryError(f'Failed to chmod ssh key file:\n{result.stderr}')
 
     def enable_rhsm_proxy(self, hostname, port=None):
-        """Configures proxy for subscription manager"""
+        """Configures HTTP proxy for subscription manager"""
         cmd = f"subscription-manager config --server.proxy_hostname={hostname}"
         if port:
             cmd += f' --server.proxy_port={port}'
+        logger.info(f'Configuring {hostname} HTTP proxy for subscription manager.')
```

Comment:
Please do either of:
1. Move the logger call before the actual action is performed or
2. Change the wording of the log message

```suggestion
        logger.info(f'Configured {hostname} HTTP proxy for subscription manager.')
```

Reply:
Well, it is running before the `self.execute(cmd)` that's why the word `configuring` and not `configured`. The first option you're suggesting.

Reply:
My bad, apologies. Resolving

Reply:
@ogajduse Do you plan to go fishing ? :fishing_pole_and_fish: